### DISCUSSION TITLE
Best Mod addon for Warehouse

### DISCUSSION THREAD

**Discussion by KalCorp** [ldtteam/minecolonies#9397] [General]
One of the best mods i use with MineColonies is Toms simple storage.

Just add a terminal on side of warehouse and have trims go inside and connect all the racks.  when upgrading you just need to put back the trim.  Just outside you can have your home base. Now you got town inventory and yours all in one.  So easy to get stuff ina and out, Even has crafting table.  and after time a wireless terminal.    Got to love it!

-KalCorp

**Comment by uecasm** [comment]
Or you can just put a Refined Storage storage bus on the warehouse block itself.  You don't need to link the racks.

Combine that with Entangled and then you don't need to repair anything after upgrading.
